lib/versioning/pvp/readme.md

```diff
@@ -0,0 +1,20 @@
+# PVP
+
+**What type of versioning is used?**
+
+> A package version number SHOULD have the form A.B.C, and MAY optionally have any number of additional components, for example 2.1.0.4 (in this case, A=2, B=1, C=0).
+> A.B is known as the major version number, and C the minor version number.
+> Specification: https://pvp.haskell.org/
+
+The one unusual difference between PVP and semver is that there are 2 major versions, and that
+there could be additional version numbers past patch.
```

Comment:
```suggestion
The one unusual difference between PVP and SemVer is that there are two major versions, and that there could be additional version numbers past _patch_.
```

This should go on one line (one sentence per line pattern).
SemVer is a specific term/name, so should be capitalized. 😉 

Are you sure you that _patch_ is the correct term? According to the Haskell Package Versioning Policy, _c_ is called _minor_.

Reply:
> SemVer is a specific term/name, so should be capitalized. 😉
Sorry I have terrible grammar. Thanks for the suggestions!

> Are you sure you that patch is the correct term? According to the Haskell Package Versioning Policy, c is called minor.

I've updated the instructions. Let me know if that's better.